config/locales/transliterate/es.yml

```diff
@@ -0,0 +1,21 @@
+es:
+  i18n:
+    transliterate:
+      rule:
+        # Convert quotation marks
+        ’: "'"
+        ‘: "'"
+        ‛: "'"
+        “: '"'
```

Comment:
what does it take for these non-EN settings to get used? We're always hardcoding `:en` in the `I18n.transliterate` call?

Reply:
Let's see if we can add an exception to `i18n-tasks` or whatever checks for equal keys? minimize our maintenance burden/headache/bug surface area